### PR TITLE
Change the way callable BED works + genotyping fixes + ped fix

### PR DESCRIPTION
**Pull request by nvnieuwk**
A small fix for when the callable BED files are given through the samplesheet. Wait for the merge until a test has been added to avoid this issue in the future. Data is under review in this [PR](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-test-datasets/pull/11)

### CODE REVIEW

**Comment by nvnieuwk** [comment]
Doesn't seem to work correctly, fix this tomorrow!

**Comment by nvnieuwk** [comment]
Fixes #104, #103, #102 and #101

**Comment by nvnieuwk** [comment]
Also fixes #64

File: conf/modules.config [hunk 1]
```diff
@@ -25,26 +25,33 @@ process {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     */
 
-    withName: "^.*:SAMPLE_PREPARATION:BEDTOOLS_INTERSECT\$" {
-        ext.prefix = {"${meta.id}_intersect"}
-        ext.args = "-sorted"
+    withName: "^.*:SAMPLE_PREPARATION:MOSDEPTH\$" {
+        ext.args = [
+            "--quantize 0:1:4 --mapq 1 --flag 1804",
+            !params.coverage ? "--no-per-base --fast-mode" : ""
```
Comment (matthdsm) [inline conf/modules.config:31]: I'd add the `--no-per-base` by default

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,20 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### New Features
 
+1. Added a `--coverage <true/false>` flag which can be used to run mosdepth in normal mode instead of fast-mode. This flag will also make sure all output from mosdepth is present in the output directory for each individual
```
Comment (matthdsm) [inline CHANGELOG.md:10]: I'm not a fan of the naming, it's not really clear what it does from sight.
Perhaps `coverage_fast` or something?

File: conf/modules.config [hunk 3]
```diff
@@ -27,18 +27,18 @@ process {
...
         publishDir  = [
             overwrite: true,
             enabled: true,
             mode: params.publish_dir_mode,
             path: { "${params.outdir}/${individuals_output}/${meta.sample}" },
             saveAs: { filename -> 
-                params.coverage ?
-                    (filename.equals('versions.yml') ? null : filename) :
-                    ((filename.endsWith(".quantized.bed.gz") || filename.endsWith(".quantized.bed.gz.csi")) && !bed ? filename : null)
+                params.coverage_fast ?
```
Comment (matthdsm) [inline conf/modules.config:39]: Perhaps remove the rule and output everything by default?

**Pull request merged**